contracts/libs/data-structures/AvlTree.sol

```diff
@@ -420,19 +422,24 @@ library AvlTree {
         function(bytes32, bytes32) view returns (int256) comparator;
     }
 
+    error NodeAlreadyExists(bytes32 key);
```

Comment:
Let's define errors by the spec at the top of the file

Reply:
okay